yarn-project/prover-client/src/orchestrator/block-building-helpers.ts

```diff
@@ -107,7 +107,7 @@ export async function buildBaseRollupInput(
     sortedNewLeavesIndexes,
   } = await db.batchInsert(
     MerkleTreeId.NULLIFIER_TREE,
-    tx.data.end.newNullifiers.map(sideEffectLinkedToNoteHash => sideEffectLinkedToNoteHash.value.toBuffer()),
+    tx.data.end.newNullifiers.map(sideEffectLinkedToNoteHash => sideEffectLinkedToNoteHash.toBuffer()),
```

Comment:
Very small nit - could this be renamed to something like `nullifier` as it's no longer a `sideEffectLinkedToNoteHash`?

Reply:
Good point!